perftest/wallet.go

```diff
@@ -123,6 +123,12 @@ func (w walletWrapper) NewMarket(offset int, user UserDetails) error {
 			"enactmentTimestamp": w.SecondsFromNowInSecs(30),
 			"newMarket": map[string]interface{}{
 				"changes": map[string]interface{}{
+					"markPriceConfiguration": map[string]interface{}{
+						"decayWeight":        "0.001",
+						"decayPower":         1,
+						"cashAmount":         "1000000",
+						"compositePriceType": 3, // LAST TRADE
```

Comment:
decayWeight and cashAmount shouldn't be specified if you're using compositePriceType. I've fixed on the latest PR, but for now it's fine (just silly of me)